catalog/lib/theme/custom_colors.dart

```diff
@@ -0,0 +1,94 @@
+// ignore_for_file: overridden_fields
+
+import 'package:flutter/material.dart';
+
+@immutable
+class CustomColors extends ThemeExtension<CustomColors> {
+  const CustomColors({
+    required this.textColor,
+    required this.info,
+    required this.success,
+    required this.warning,
+    required this.danger,
+  }) : super();
+
+  final Color? textColor;
+  final Color? info;
+  final Color? success;
+  final Color? warning;
+  final Color? danger;
+
+  static CustomColors getCustomColors() => const CustomColors(
```

Comment:
Is this necessary, isn't it accessible by `Theme.of(context).extension<MyColors>()!`

Reply:
Would we like to access the colors that way?